app/src/main/java/com/shagalalab/constitution/di/Injections.kt

```diff
@@ -23,8 +24,12 @@ val dataModule = module {
     single { get<ConstitutionDatabase>().partDao() }
 }
 
+val module = module {
```

Comment:
It is better to give a more specific names, maybe we can say `executorModule` (and change it if other declaration will be added later).